core/src/main/java/org/commonjava/maven/ext/core/impl/RelocationManipulator.java

```diff
@@ -308,6 +326,36 @@ private boolean updatePlugins( final WildcardMap<ProjectVersionRef> relocations,
         return result;
     }
 
+    /**
+     * This handles updating a value (be it from a Dependency or Plugin) e.g. {@code artifactId} to its new value taking into
+     * account if there are any properties.
+     *
+     * @param project a references to the Maven Project
+     * @param originalValue the original property value e.g. the value of {@code artifactId}
+     * @param originalRelocation the complete original relocation
+     * @param relocation the portion of the relocation that is currently being processed
+     * @param c a Consumer function that will update the corresponding Dependency/Plugin if its not a property.
+     * @throws ManipulationException if an error occurs.
+     */
+    private void updateString( Project project, String originalValue, ProjectVersionRef originalRelocation, String relocation,
+                               Consumer<String> c ) throws ManipulationException
+    {
+        if ( StringUtils.contains( originalValue, "$" ))
```

Comment:
Only a style comment, but in other code recently I used a static import for any `StringUtils` methods.

Reply:
Pros/cons ... I like to make it obvious where its coming from.

Reply:
That's fine either way, but maybe we should have enforced a single style across the project.

Reply:
True we don't really specify that :-( Happy to discuss offline ; pros/cons to both approaches.